docs/source/swift-scripting.md

```diff
@@ -124,87 +123,84 @@ One of the convenience wrappers available to you in the target is `ApolloSchemaD
       exit(1)
     }
     ```
-    Note that `catch`'ing and manually calling `exit` with a non-zero code leaves you with a much more legible error message than simply letting the method throw. 
+    Note that `catch`ing and manually calling `exit` with a non-zero code leaves you with a much more legible error message than simply letting the method throw. 
 
-5. Build and run using the Xcode project. Note that if you're on Catalina you may get a warning asking if your executable can access files in a particular folder like this:
+5. Build and run using the Xcode project. Note that if you're on Catalina you might get a warning asking if your executable can access files in a particular folder like this:
 
    ![permission prompt](screenshot/would_like_to_access.png)
    
-   Click the "OK" button. Your CLI output will look something like this: 
+   Click **OK**. Your CLI output will look something like this: 
    
-   ![log barf for successful run](screenshot/schema_download_success.png)
+   ![log output for successful run](screenshot/schema_download_success.png)
    
-   Those last two lines - "Saving schema started" and "Saving schema completed" indicate that the schema has successfully downloaded. 
+   The last two lines  (`Saving schema started` and `Saving schema completed`) indicate that the schema has successfully downloaded. 
+
+Note the warning: This isn't relevant for schema downloading, but it *is* relevant for generating code: In order to generate code, you need both the schema and some kind of operation.
 
-Note the warning: This isn't relevant for schema downloading, but it *is* relevant for generating code: In order to generate code, you need both the schema and some kind of operation. Now that you've got the schema, it's time to 
+## Using codegen to create a `.graphql` file with an operation
 
-## Codegen theory + creating a `.graphql` file with an operation
+Code generation requires both of the following to run:
 
-Code generation takes a combination of the **schema**, which defines what it's *possible* for you to request from or send to your server, and your **operations**, which define what you are *actually* requesting from the server. 
+* Your **schema**, which defines what it's *possible* for you to request from or send to your server
+* A collection of **operations**, which define what you are *actually* requesting from the server
```

Comment:
`collection of operations` makes it sound like there should be more than one when we need *at least one*. Any thoughts on how we could phrase that more clearly? Maybe `non-empty collection`?